db/seeds.rb

```diff
@@ -5,16 +5,33 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'nokogiri'
+require 'open-uri'
 
-Card.create(original_text: "Fine",
-            transcription: "[faɪn]",
-            translated_text: "Сухой (о погоде), хороший, штраф",
-            review_date: "10-10-2015")
-Card.create(original_text: "Meet / Met / Met",
-            transcription: "[miːt] / [met] / [met]",
-            translated_text: "Встречать, собираться, видеться",
-            review_date: "10-10-2015")
-Card.create(original_text: "Phone",
-            transcription: "[fəʊn]",
-            translated_text: "Телефон, телефонная трубка, звонить по телефону",
-            review_date: "10-10-2015")
+wiki_doc = Nokogiri::HTML(
+  open('https://en.wikipedia.org/wiki/Most_common_words_in_English')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.